### ISSUE TITLE
Our first project

### ISSUE THREAD

**Issue by mikealexander** [crumblware/meta#2]
Lunchcast? Or something else?

**Comment by mikealexander** [comment]
cc @CrumbleDynamics/development-team

**Comment by gsmithu** [comment]
+1 Lunchcast.

**Comment by sdmadden** [comment]
Isn't lunchcast already a thing, though? Like "Down To Lunch". I could see a dedicated super simple version being good, though.

**Comment by mikealexander** [comment]
@sdmadden yeah it's definitely already been done. We can def make it better and more simple imo.
